aos_cd_jobs/updater.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+from os import getenv, listdir, rename, walk
+from os.path import join, relpath
+from shutil import rmtree
+
+from git import Repo
+
+from aos_cd_jobs.common import JOBS_DIRECTORY, initialize_repo
+
+def update_branches(repo):
+    for job in list_jobs(repo):
+        if job in repo.branches:
+            branch = repo.branches[job]
+        else:
+            branch = None
+        if branch is not None:
```

Comment:
I'm still not really sure we need this -- the example I had from before was:

```py
if job in repo.branches:
    repo.branches[job].delete()
create_remote_branch(repo, job)
```

Still keeps us from having to do `None` checks, etc